### DISCUSSION TITLE
[Bug] The github-actions bot is not auto updating the app manifests in MyScoop bucket.

### DISCUSSION THREAD

**Discussion by kris3713** [ScoopInstaller/Scoop#5175] [Help and Questions]
#### Current Behavior

In my [scoop bucket](https://github.com/Zliced13/MyScoop), the **github-actions** bot is not auto updating my manifests. What am I doing wrong?

#### Expected Behavior

The **github-actions** bot should autoupdate my app manifests.

### System details

**Windows version:** [e.g. 7, 8, 10] 
10

**OS architecture:** [e.g. 32bit, 64bit]
64bit

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
7.2.6

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
{
    "lastUpdate":  "2022-10-07T08:30:08.1345008-06:00",
    "SCOOP_REPO":  "https://github.com/ScoopInstaller/Scoop",
    "SCOOP_BRANCH":  "master",
    "aria2-warning-enabled":  false,
    "aria2-enabled":  true
}
```

**Comment by kris3713** [comment]
Can anyone help me please? I will literally take anyone's help for this issue.

**Comment by rashil2000** [comment]
Have you manually enabled the Excavator workflow on Actions page?

**Reply by kris3713** [reply]
I have already. This is what I get 
https://github.com/Zliced13/MyScoop/actions/runs/3205736065

**Comment by rashil2000** [comment]
Delete this file https://github.com/Zliced13/MyScoop/blob/master/.github/CODEOWNERS and try again.

**Comment by kris3713** [comment]
It turns out I needed to enable **Allow all actions and reusable workflows** in my repo settings. Now when I run the excavator, it's successful.
![(firefox)b7s479_10-07-2022](https://user-images.githubusercontent.com/101912712/194590935-9688b61c-e638-4bd3-b8cb-f51818c514e0.jpg)
![(%pn)-I0M5v0_10-07-2022](https://user-images.githubusercontent.com/101912712/194591262-417d12cd-458c-43b1-8d03-1e7bd26a7ecb.jpg)
